marda_registry/data/extractors/yadg.yml

```diff
@@ -51,7 +51,18 @@ citations:
 source_repository: https://github.com/dgbowl/yadg
 documentation: https://dgbowl.github.io/yadg
 usage:
-    - cli:yadg extract {{ file_type }} {{ file_path }}
+    - method: cli
+      command: yadg extract {{ input_type }} {{ input_path }} {{ output_path }}
+    - method: python
+      setup: yadg
+      command: yadg.extractors.extract({{ input_path }}, {{ input_path }})
+installation:
+    - method: pip
+      packages:
+          - yadg>=5.0a2
+    - method: pip
+      packages:
+          - git+https://github.com/dgbowl/yadg.git@5.0a2
```

Comment:
This isn't blocking but I wonder if we should restrict to one set of instructions per method? I know it makes sense here as 5.0a2 isn't on PyPI, but it will be hard to resolve which to use automatically

Reply:
Hmm, I thought we might iterate through those until one of the installation methods works.